app/views/credit_cards/show.html.erb

```diff
@@ -21,6 +21,10 @@
   <% end %>
   <% @reservations.each do |reservation| %>
     <div class= "py-4">
+      <span class="badge rounded-pill text-bg-info my-2"><%= reservation.status %></span>
+      <% if @credit_card.user_id == current_user.id %>
+        <p><%= link_to "Accept", accept_path(reservation), data: { turbo_method: :patch} %> | <%= link_to "Reject", reject_path(reservation), data: { turbo_method: :patch} %></p>
```

Comment:
If it gets a bit lengthy, we can make this a span line for more readability, but I'm fine this way.